src/docker/compose.yaml

```diff
@@ -81,12 +81,21 @@ services:
     image: 'econialabs/emojicoin-dot-fun-indexer-processor'
     container_name: 'processor'
     healthcheck:
-      test: 'curl -f http://localhost:${PROCESSOR_WS_PORT}/ || exit 1'
+      # yamllint disable rule:line-length
+      test:
+      - 'CMD-SHELL'
+      - |
+          g="indexer_processor_success_count{processor_name=\"emojicoin_processor\"}"
+          res=$(curl -s http://localhost:8084/metrics)
+          success_count=$(echo "$$res" | grep "$$g")
+          count=$(echo "$$success_count" | awk '{ print $2 }' | xargs)
+          test "$${count:-0}" -gt 0 && curl -sf http://localhost:${PROCESSOR_WS_PORT}
```

Comment:
Change to polling check per eng call, instead of healthcheck, since this isn't totally accurate

Reply:
Done